### PR TITLE
[dev-tps-warping] Apply PEP08 guidelines

### PR DESCRIPTION
**Pull request by decorouz**
## Description
Apply PEP08 guidelines to import statements and character lengths.

## Checklist

- A descriptive but concise pull request title
- [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [Unit tests](https://scikit-image.org/docs/dev/development/contribute.html#testing)
- A gallery example in `./doc/examples` for new features
- [Contribution guide](https://scikit-image.org/docs/dev/development/contribute.html) is followed

### CODE REVIEW

**Comment by lagru** [comment]
Thanks @decorouz, I'll have a look. But please also tag @ana42742 next time so that she is aware of your changes. 

I am thinking that to be polite, we should add the prefix "[dev-tps-warping]" to PRs that target the `dev-tps-warping`. That way other maintainers see immediately that this is not targeting `main`.

File: skimage/transform/_thin_plate_splines.py [hunk 1]
```diff
@@ -1,11 +1,15 @@
+import numpy as np
 from scipy import ndimage
-import numpy
 
-def warp_images(from_points, to_points, images, output_region, interpolation_order = 1, approximate_grid=2):
+
+def warp_images(
+        from_points, to_points, images, output_region, interpolation_order=1,
+        approximate_grid=2):
```
Comment (lagru) [inline skimage/transform/_thin_plate_splines.py:7]:
```suggestion
def warp_images(
    from_points,
    to_points,
    images,
    output_region,
    interpolation_order=1,
    approximate_grid=2
):
```

**Comment by lagru** [comment]
Linting job passed. So merging as there are no tests yet.

**Comment by decorouz** [comment]
> Thanks @decorouz, I'll have a look. But please also tag @ana42742 next time so that she is aware of your changes.
> 
> I am thinking that to be polite, we should add the prefix "[dev-tps-warping]" to PRs that target the `dev-tps-warping`. That way other maintainers see immediately that this is not targeting `main`.

Great suggestion. @ana42742 take note.
Thanks @lagru

**Pull request merged**